### PR TITLE
UI: Add search button to search bar

### LINKED ISSUE #529: [suggestion] search button
it would be great to have a button that starts the search next to the search box. in some situations a pointing device is the easier choice and a button would allow this way of usage. maybe put it inside the text field like it is done with the firefox search box

### PR DESCRIPTION
**Pull request by thansk**
Closes #529

### CODE REVIEW

**Comment by unixfox** [comment]
Why?

**Comment by ChunkyProgrammer** [comment]
Related: https://github.com/iv-org/invidious/pull/4098/

**Comment by thansk** [comment]
Because I find that sometimes I enter something into the box and later want to submit it without my keyboard around. 

I think it's generally a good idea for accessibility to make invidious usable without a keyboard.

**Comment by unixfox** [comment]
Ha ok you just added a button on the search bar.

The title is very confusing.

To be fair I quite like the way it is without any search button.

I think I good middle ground would be to just have the search icon. Not the full text "Search". Like on youtube.com

**Comment by thansk** [comment]
I added an icon instead and used the aria-label for it with the translation like @ChunkyProgrammer suggested. Icon is using the currentColor so it works on dark and light theme. It's MIT licensed from heroicons.

**Comment by unixfox** [comment]
Please use the built-in icons instead: https://github.com/iv-org/invidious/blob/master/assets/fonts/ionicons.svg

**Comment by thansk** [comment]
Yeah, that makes sense, it should be good now.

**Review by SamantazFox — APPROVED** [review]

**Review by unixfox — APPROVED** [review]

File: assets/css/default.css [hunk 1]
```diff
@@ -278,7 +278,14 @@ div.thumbnail > .bottom-right-overlay {
   display: inline;
 }
 
-.searchbar .pure-form fieldset { padding: 0; }
+.searchbar .pure-form {
+  display: flex;
+}
+
+.searchbar .pure-form fieldset {
```
Comment (SamantazFox) [inline assets/css/default.css:285]:
```suggestion
.searchbar .pure-form fieldset {
```

**Comment by SamantazFox** [comment]
Thanks for contributing to invidious :D

**Pull request merged**